cvat/apps/dataset_manager/bindings.py

```diff
@@ -1863,6 +1863,8 @@ def load_dataset_data(project_annotation, dataset: dm.Dataset, project_data):
     else:
         for label in dataset.categories()[dm.AnnotationType.label].items:
             if not project_annotation.db_project.label_set.filter(name=label.name).exists():
+                if label.name == 'background':
```

Comment:
This seems like a questionable fix to me. What if there are items with annotations that contain this label, but the CVAT project doesn't have it? At the very least, I think we should only ignore a label if it's unused.

Reply:
Fixed